client/src/App.scss

```diff
@@ -131,6 +160,12 @@ UserList
   font-weight: normal;
 }
 
+.usersViewBtns {
+  display: flex;
+  justify-content: space-around;
+  width: 40%;
+  margin-bottom: 2 * $unit;
+}
```

Comment:
same thing as the login buttons here, i think these can just be styled with the same reused classes